shared/src/common/enums/validation/add-episode-validation-rule.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum AddEpisodeValidationRule {
```

Comment:
rename `shared/src/common/enums/validation/episode-validation-rule.enum.ts`
```suggestion
enum EpisodeValidationRule {
```

Reply:
fixed it in next commit